app/src/interfaces/_system/system-raw-editor/system-raw-editor.vue

```diff
@@ -43,7 +47,7 @@ onMounted(async () => {
 		CodeMirror.defineSimpleMode('mustache', mustacheMode);
 
 		codemirror = CodeMirror(codemirrorEl.value, {
-			mode: 'mustache',
+			mode: props.language === undefined ? 'mustache' : props.language,
```

Comment:
Let's turn `mustache` into a default value instead for the language prop (line 30) as it feels like that'd indicate the default mode better. Seems like `input-code` interface also does the same default value usage as `plaintext`, so let's do the same here but with `mustache`.